src/Test/Message.elm

```diff
@@ -32,9 +32,9 @@ failureMessage { given, description, reason } =
 
         Invalid BadDescription ->
```

Comment:
Didn't we trim whitespace? Are there really two cases right below here then? If so, and the only case when it is not an empty string is when it is only whitespace characters, could we put some quotes around the description? 

Otherwise you might get the `This is an invalid test description:` error message, with a description that looks like an empty string, which means you immediately search for `""`, which gives no hits, because it was actually named `" "`.